_data/simple-icons.json

```diff
@@ -11624,6 +11624,12 @@
             "hex": "FFBD16",
             "source": "https://picrew.me"
         },
+        {
+            "title": "Picsart",
+            "hex": "C209C1",
+            "source": "https://commons.wikimedia.org/wiki/File:Picsart_(software_company)_logo.svg",
```

Comment:
Let's use the one from their header instead.

Reply:
> Let's use the one from their header instead.

Sorry if I was kind of unclear, the hex from their header is the same as the one in Commons, it's the hex in their brand guidelines that's different.

Reply:
No I mean the source file. Generally we like to go with the company/products official icon rather than a 3rd party source, though if say the company's logo is a PNG we can use the Wikimeda Commons version instead if it matches.

Reply:
Oh, right.
Well, the source file's hex is the same as Commons' is what I tried to say. Sorry for any misunderstanding.

Reply:
```suggestion
            "source": "https://picsart.com",
```

Reply:
As I mentioned in the PR's opening message, the vector I used is from Commons, & it's slightly different from the one on the website, would I have to rectify the vector to the one that is on the website?

Reply:
Yes, this is why we use the source icon rather than Wikimedia Commons since they aren't always fully accurate.

Reply:
Oh wait, nevermind, I double-checked & I did in fact use the vector from the website, so it's all good.